### PR TITLE
server: fix NPE when router.service.offering is set due to service/disk offering refactoring

### PR DESCRIPTION
**Pull request by weizhouapache**
### Description

This PR fixes NPE by initializing diskOfferingDao.

```
java.lang.NullPointerException
        at org.cloud.network.router.deployment.RouterDeploymentDefinition.verifyServiceOfferingByUuid(RouterDeploymentDefinition.java:411)
        at org.cloud.network.router.deployment.RouterDeploymentDefinition.findAccountServiceOfferingId(RouterDeploymentDefinition.java:400)
        at org.cloud.network.router.deployment.RouterDeploymentDefinition.findServiceOfferingId(RouterDeploymentDefinition.java:423)
        at org.cloud.network.router.deployment.RouterDeploymentDefinition.executeDeployment(RouterDeploymentDefinition.java:378)
        at org.cloud.network.router.deployment.RouterDeploymentDefinition.findOrDeployVirtualRouter(RouterDeploymentDefinition.java:242)
        at org.cloud.network.router.deployment.RouterDeploymentDefinition.deployVirtualRouter(RouterDeploymentDefinition.java:208)

```

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by nvazquez** [comment]
@blueorangutan test

File: server/src/main/java/org/apache/cloudstack/network/router/deployment/VpcRouterDeploymentDefinition.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
-package org.cloud.network.router.deployment;
+package org.apache.cloudstack.network.router.deployment;
```
Comment (nvazquez) [inline server/src/main/java/org/apache/cloudstack/network/router/deployment/VpcRouterDeploymentDefinition.java:17]: Not blocking question: what's the reason for these package changes?

**Review by nvazquez — APPROVED** [review]
Code LGTM

File: server/src/main/java/org/apache/cloudstack/network/router/deployment/VpcRouterDeploymentDefinition.java (hunk 1 above)
Comment (weizhouapache) [reply server/src/main/java/org/apache/cloudstack/network/router/deployment/VpcRouterDeploymentDefinition.java:17]: @nvazquez 
in cloudstack codebase, old java classes are in package `com.cloud.xx.xx` ,  new java classes are in package `org.apache.cloudstack.xx.xx`

these classes are in packages `org.cloud.xx.xx` which are not correct I think. therefore move to `org.apache.cloudstack`

**Review by DaanHoogland — APPROVED** [review]
Looks good

**Pull request merged**